Readme.md

```diff
@@ -46,15 +46,95 @@ If you'd like to build your own VFS for Git Windows installer:
+
+* Ensure you have Xcode installed and have accepted the terms of use and launched Xcode at least once.
+* Install [Visual Studio for Mac ](https://visualstudio.microsoft.com/vs/mac). (This will also install the `dotnet` CLI).
+* If you still do not have the `dotnet` cli `>= v2.1.300` installed [manually install it](https://www.microsoft.com/net/download/dotnet-core/2.1)
+* [Install Homebrew](https://brew.sh/)
+* Install Java with
+  ```
+  brew cask install java
+  ```
+
+* Install the Git credential manager for Mac [by following their Homebrew instructions](https://github.com/Microsoft/Git-Credential-Manager-for-Mac-and-Linux/blob/master/Install.md#installing-on-mac-using-homebrew-or-on-linux-using-linuxbrew-recommended)
+
+* If you are not currently an Apple Developer you will need to become one so that you can either use Microsoft certs or create your own certs for signing purposes. (If you are a Microsoft employee you must use your alias@microsoft.com when creating your Apple account.)
+
+* Create a `VFS` directory and Clone the VFS repo into a directory called `src` inside it:
+  ```
+  mkdir VFS && cd VFS && git clone https://github.com/Microsoft/VFSForGit.git src && cd src
+  ```
+
+* Disable "System Integrity Protection" (for loading unsigned kexts) by booting into recovery mode (`[Win/⌘] + R` while booting).
+  Once booted into recovery mode open Utilities -> Terminal to launch a termnial. Enter:
+  ```
+  csrutil disable
+  ```
+  to disable SIP.
+
+* Now `cd` back into your VFS src directory and run
+
+  ```
+  Scripts/Mac/BuildGVFSForMac.sh [Release]
+  ```
+
+  **Troubleshooting**
+
+  If you get
+  ```
+  xcodebuild: error: SDK "macosx10.13" cannot be located.
+  ```
+  You may have the "XCode Command Line Tools" installed (helpfully by Mac OS) instead of full XCode.
+  Make sure
+  ```
+  xcode-select -p
+  ```
+
+  shows `/Applications/Xcode.app/Contents/Developer`. If it does not, install Xcode and then launch it (you can close it afterwards.)
+
+* Now setup the git credential manager by running
+
+  ```
+  Scripts/Mac/PrepFunctionalTests.sh
+  ```
+
+* Now you have to load the ProjFS Kext.
+
+  ```
+  ProjFS.Mac/Scripts/LoadPrjFSKext.sh [Release]
+  ```
+
+* Now you can put your built gvfs program on your path. A simple way to do that is by adding `Path/to/VFS/Publish` to your path.
+
+  Check you have it by running
+
+  ```
+  command -v gvfs
+  ```
+
+  You should see a path to the gvfs executable.
+
+* Try cloning! Now that you have `gvfs` ready you can try cloning a VFS for Git enabled repository!
+
+  ```
+  gvfs clone URL_TO_REPOSITORY [--cache-url] --local-cache-path ~/.gvfsCache
+  ```
+
+  Note you may have a cache server URL to use. Note the current use of `--local-cache-path`. That argument prevent gvfs from running into a permissions error trying to put the cache at the root of your Mac's hard-drive.

```

Comment:
Seems out of scope for people who are interested in hacking on the VFS For Git codebase rather than customers who want to run a VFS for Git repo on their machine.